src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+// Перевіряємо, чи існує таблиця
+if (table) {
+  const rows = Array.from(table.rows);
+
+  // Створюємо DocumentFragment для уникнення переробок DOM
+  const fragment = document.createDocumentFragment();
+
+  rows.forEach((row) => {
+    const secondCell = row.cells[1]; // Отримуємо другу клітинку
+
+    if (secondCell) {
+      const clonedCell = secondCell.cloneNode(true); // Клонуємо другу клітинку
+
+      // Вставляємо клон перед останньою клітинкою
+      row.insertBefore(clonedCell, row.cells[row.cells.length - 1]);
```

Comment:
You are inserting the cloned cell before the last cell of the row. However, it is not clear why you are doing this. If you want to add a cell at the end of the row, you can simply use the `appendChild` method on the `row`.